src/guides/v2.3/get-started/gs-curl.md

```diff
@@ -28,6 +28,119 @@ Option | Description
 `-T` | Transfers the specified local file to the remote URL.
 `-X` | Specifies the request method to use when communicating with the HTTP server. The specified request method is used instead of the default GET method.
 
+### Using cUrl in Magento
+
+Magento provides native functionality for cURL instead of using default php cURL. The class ``Magento\Framework\HTTP\Client\Curl`` can be used to work with HTTP protocol using curl library.
+At first, the instance of ``Magento\Framework\HTTP\Client\Curl`` should be created.
+
+```php
+/**
+* Constructor.
+*
+* @param Magento\Framework\HTTP\Client\Curl $curl
+*/
+public function __construct(
+   Magento\Framework\HTTP\Client\Curl $curl
+) {
+   $this->_curl = $curl;
```

Comment:
Please, remove the underscore from the property name, so we have `$this->curl`.  PSR-2 is really strict about that:

> Property names SHOULD NOT be prefixed with a single underscore to indicate protected or private visibility.

Reply:
This recommendation also applies to all other examples